src/neptune_airflow/impl/__init__.py

```diff
@@ -223,7 +223,8 @@ def _log_context(context: Dict[str, Any], neptune_run: Union[Run, Handler]) -> N
 
                     neptune_run[f"context/{field}/inversed_deprecated_options/{key}"] = stringify_unsupported(value)
 
-            neptune_run[f"context/{field}"] = stringify_unsupported(to_log.__dict__)
+            to_log_dict = copy(to_log.__dict__)
```

Comment:
`stringify_unsupported()` does not modify any values. Is this needed?

Reply:
This `__dict__` sometimes changes size in the runtime (afaik nothing relevant to airflow per se). Since `stringify_unsupported` iterates over this dict, python complains with:
`RuntimeError: dictionary changed size during iteration`